Add Google and Apple login buttons, fix a couple of typos in app strings

littleangelx opened the pull request:
Added sample Google and Apple login buttons - just the buttons with no functionality.

Also fixed a couple of typos in the App strings file.

![Screenshot_1740504600](https://github.com/user-attachments/assets/48496d5a-4190-414a-b192-68d917c7ac24)

marksheekey commented:
Unfortunately you cant just use your own font/styling for the social sign in buttons.
Google...
https://developers.google.com/identity/branding-guidelines
I'm sure Apple will be the same.

This looks a nice pacakge to make things easier.
https://pub.dev/packages/sign_in_button

fion21 commented:
it's good Rachel, thank you for your effort, stlying wise please increase the font size, up 2 o3 3 more and as it's white it needs to be  bold as well. Have you done a safe area wrapper?

littleangelx commented:
Thanks Mark and Fiona. Sorry, I was unaware of the brand guidelines - I will look into making the buttons correctly.

Regarding things like specifying the height, I copied that basic code for the elevated button from code that already exists in the project (for the primary button), which was written by someone else. I wanted to make it fit with the already created login button. I just changed the background colour and added the images.

Thanks again and I will try to sort it using the package you suggest Mark.

fion21 commented:
Approved
